lib/mockbuilder.py

```diff
@@ -17,6 +17,7 @@
 
```

Comment:
It seems to me that, with this approach, if you desire to build gcc and 2 packages which depend on gcc, gcc will be built 3 times. This does not make sense to me. Maybe I misunderstood? 

I don't think we need to keep the built build dependencies in a package-specific directory, I'd prefer to have a directory for each package built (named after a primary key) and keep them all under a single directory, as mentioned in the other patch of this PR.